api/crds/v1alpha1/restic.yaml

```diff
@@ -1771,6 +1771,15 @@ spec:
                     type: string
                   prune:
                     type: boolean
+                required:
+                - name
+                - keepLast
```

Comment:
- keepLast
                - keepHourly
                - keepDaily
                - keepWeekly
                - keepMonthly
                - keepYearly

are not required.